example-client/client.py

```diff
@@ -64,43 +64,76 @@ def run_structure(input: str) -> Optional[str]:
         args=[input],
     )
 
-    # Runs are asynchronous, so we need to poll the status until it's no longer running.
     structure_run_id = structure_run["structure_run_id"]
-    status = structure_run["status"]
-    printed_event_ids = set()  # Keep track of which events we've printed.
-    while status not in ("SUCCEEDED", "FAILED"):
-        structure_run = get_structure_run(
-            host=HOST, api_key=GT_API_KEY, run_id=structure_run_id
-        )
-        status = structure_run["status"]
 
-        # You can comment out this block if you don't want to stream events as they occur.
+    received_event_ids = set()  # Keep track of which events we've already received.
```

Comment:
Could/should be replaced with [some form of pagination](https://betterprogramming.pub/understanding-the-offset-and-cursor-pagination-8ddc54d10d98).